### ISSUE TITLE
[storagenode] Monitor and Enable lazyfilewalker by default

### ISSUE THREAD

**Issue by profclems** [storj/storj#5861]
**Describe the feature or problem you'd like to solve**

The lazyfilewalker runs the used-space and garbage collection filewalkers as a low priority subprocess. It came up as a solution to https://github.com/storj/storj/issues/5349, where these filewalkers run with the same I/O priority as customer downloads.

The lazyfilewalker is disabled by default.

**What needs to be done:**

- Enable the lazyfilewalker on a production node. Maybe ask SNOs to volunteer on the forum?
- Monitor and collect profiles and monkit logs
- Enable the lazyfilewalker by default

**Comment by iglesiasbrandon** [comment]
We will ask the SNO community to volunteer to test out this functionality to determine if they have any issues.

**Comment by profclems** [comment]
Logs from an instance of the gc-filewalker:

```
05:06:14 root@testnode ~ → docker logs storagenode | grep lazyfilewalker.gc-filewalker
2023-05-25T17:00:22.139Z	INFO	lazyfilewalker.gc-filewalker	starting subprocess	{"process": "storagenode", "satelliteID": "1wFTAgs9DP5RSnCqKV1eLf6N9wtk4EAtmN5DpSxcs8EjT69tGE"}
2023-05-25T17:00:22.142Z	INFO	lazyfilewalker.gc-filewalker	subprocess started	{"process": "storagenode", "satelliteID": "1wFTAgs9DP5RSnCqKV1eLf6N9wtk4EAtmN5DpSxcs8EjT69tGE"}
2023-05-25T17:00:22.166Z	INFO	lazyfilewalker.gc-filewalker.subprocess	Database started	{"process": "storagenode", "satelliteID": "1wFTAgs9DP5RSnCqKV1eLf6N9wtk4EAtmN5DpSxcs8EjT69tGE", "process": "storagenode"}
2023-05-25T17:00:22.168Z	INFO	lazyfilewalker.gc-filewalker.subprocess	gc-filewalker started	{"process": "storagenode", "satelliteID": "1wFTAgs9DP5RSnCqKV1eLf6N9wtk4EAtmN5DpSxcs8EjT69tGE", "process": "storagenode", "createdBefore": "2023-04-18T18:08:51.963Z", "bloomFilterSize": 239182}
2023-05-25T17:00:22.172Z	INFO	lazyfilewalker.gc-filewalker.subprocess	gc-filewalker completed	{"process": "storagenode", "satelliteID": "1wFTAgs9DP5RSnCqKV1eLf6N9wtk4EAtmN5DpSxcs8EjT69tGE", "process": "storagenode", "piecesCount": 67, "piecesSkippedCount": 0}
2023-05-25T17:00:22.176Z	INFO	lazyfilewalker.gc-filewalker	subprocess finished successfully	{"process": "storagenode", "satelliteID": "1wFTAgs9DP5RSnCqKV1eLf6N9wtk4EAtmN5DpSxcs8EjT69tGE"}
```

**Comment by profclems** [comment]
I created a new issue from a bug on windows reported on the forum post https://github.com/storj/storj/issues/6006

**Comment by profclems** [comment]
Blocked by #6006

**Comment by iglesiasbrandon** [comment]
#6006 has been merged but not deployed so this ticket is on pause
